server/v2/appmanager/mock/storage.go

```diff
@@ -1,34 +1,9 @@
 package mock
 
 import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
 	"cosmossdk.io/log"
-	storev2 "cosmossdk.io/store/v2"
-	"cosmossdk.io/store/v2/commitment"
-	"cosmossdk.io/store/v2/commitment/iavl"
-	"cosmossdk.io/store/v2/db"
-	"cosmossdk.io/store/v2/storage"
-	"cosmossdk.io/store/v2/storage/pebbledb"
 )
 
-func StateCommitment(_ *testing.T) storev2.Committer {
```

Comment:
removed for now since we dont use ss and sc anymore in the app. we just assume a database